src/helix-ui/elements/HXFormControlElement.js

```diff
@@ -1,9 +1,9 @@
 import { HXElement } from './HXElement';
 
-const STATE = {
-    changed: 'hx-changed',
+export const STATE = {
+    changed: 'hx-change',
```

Comment:
Please revert. 

Attributes should describe the element (i.e., adjective: "_changed_ thing"), rather than communicate an event/action (i.e., verb: "thing _changed_").

Reply:
reverted change